src/codegen/build.sbt

```diff
@@ -1,4 +1,5 @@
 //> DependsOn: core
+//> DependsOn: fuzzing
```

Comment:
This is so @mabusch can add python generated tests to codegen. Do you want to commit the python test generation to spark as well? If so it will need to depend on fuzzaing in some way. Alternatively python testing can be done in fuzzing